withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -0,0 +1,99 @@
+# Withdraw by check JSON contract
+To get things started, the client will send an initial request structured like so:
+```json
+{
+  "rgState": "STATESTART",
+  "powerOnFileName": "BANNO.WITHDRAW.CHECK.V1.POW",
+  "userChrList":[
+    {"id": 1, "value": "member-account-number"},
+    {"id": 2, "value": "4charLoanId"},
+    {"id": 3, "value": ""},
+    {"id": 4, "value": ""},
+    {"id": 5, "value": ""}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+If successful, the poweron should respond with:
+```json
+{
+  "results": {
+    "shareEligible": true,
+    "accountNumber": "0123456789",
+    "accountDescription": "ADVANCED CHECKING",
```

Comment:
Just to clarify... is this the ACCOUNT:DESCRIPTION or [SHARE/LOAN]:DESCRIPTION

Reply:
I've updated it to be `shareLoanDescription` for clarity.